packages/reassure-compare/src/utils/format.ts

```diff
@@ -99,8 +99,8 @@ function getRenderCountSymbols(entry: CompareEntry) {
 }
 
 export function formatMetadata(metadata?: PerformanceMetadata) {
-  if (metadata?.branch && metadata?.commitHash) {
-    return `${metadata.branch} (${metadata.commitHash})`;
+  if (metadata?.branch && metadata?.commitHash && metadata.creationDate) {
```

Comment:
Creation date is optional, older files will not contain it, therefor you should modify the code to support formatting in both cases with and without creation date